doc/document/PortingToFreeRTOSAWS.md

```diff
@@ -0,0 +1,298 @@
+# FreeRTOS AWS porting guide

```

Comment:
Is ```FreeRTOS AWS``` a proper noun?

Reply:
I got this proper noun from https://github.com/aws/amazon-freertos.
Then the title should be ```The porting guide for FreeRTOS AWS```